Resources/Prototypes/Entities/Objects/Fun/Tabletop/nanopoliafigurines.yml

```diff
@@ -809,72 +785,38 @@
 
 
 - type: entity
-  parent: BaseItem
+  parent: Card_Dispo_Train1
```

Comment:
again for all these dont use _ in the ids

Reply:
Fixed now for sure

Reply:
> Now Fixed for sure

Okay, I was in a hurry before checking for errors and said this